### PR TITLE
kind 0.11.1

### PR DESCRIPTION
**Pull request by sestegra**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/kind.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Kind < Formula
   desc "Run local Kubernetes cluster in Docker"
   homepage "https://kind.sigs.k8s.io/"
-  url "https://github.com/kubernetes-sigs/kind/archive/v0.11.0.tar.gz"
-  sha256 "c78bed3a39afd2e523b38256b7f16445d0ad4b8b25f18b5766005e095af772cd"
+  url "https://github.com/kubernetes-sigs/kind/archive/refs/tags/v0.11.1.tar.gz"
```
Comment (nandahkrishna) [inline Formula/kind.rb:4]:
```suggestion
  url "https://github.com/kubernetes-sigs/kind/archive/v0.11.1.tar.gz"
```
We can use the shorter URL here.

Comment (sestegra) [reply Formula/kind.rb]: Done

**Comment by SMillerDev** [comment]
Could you update the `install` block to use the new style?
```patch
-    system "go", "build", "-o", bin/"kind"
-    prefix.install_metafiles
+   system "go", "build", *std_go_args
```

**Review by nandahkrishna — APPROVED** [review]
Thanks, @sestegra!

**Pull request closed without merging**